### DISCUSSION TITLE
use of Astra toolbox for industrial CT

### DISCUSSION THREAD

**Discussion by asyali-mh** [astra-toolbox/astra-toolbox#437] [Q&A] [answered]
Hello Astra Users!
Can we use Astra toolbox for industrial CT (source & sensor fixed, object rotating) applications? If yes, how can we define the required projection geometry?
Thanks..

**Comment by askorikov** [comment, marked as answer]
Hi! The geometry you describe should be computationally equivalent to the source & detector rotating, object fixed convention of Astra (at least for simple circular geometries), so I think you can just use normal scripts provided in the examples folder.

**Reply by asyali-mh** [reply]
Thank you very much..
